src/terminal/terminal.c

```diff
@@ -2224,3 +2254,80 @@ void guac_terminal_remove_user(guac_terminal* terminal, guac_user* user) {
     /* Remove the user from the terminal cursor */
     guac_common_cursor_remove_user(terminal->cursor, user);
 }
+
+bool guac_terminal_is_part_of_word(int ascii_char) {
+    if ((ascii_char >= '0' && ascii_char <= '9') || 
+        (ascii_char >= 'A' && ascii_char <= 'Z') || 
+        (ascii_char >= 'a' && ascii_char <= 'z') ||
+        (ascii_char == '$') ||
+        (ascii_char == '%') ||
+        (ascii_char == '&') ||
+        (ascii_char == '-') ||
+        (ascii_char == '.') ||
+        (ascii_char == '/') ||
+        (ascii_char == ':') ||
+        (ascii_char == '=') ||
+        (ascii_char == '?') ||
+        (ascii_char == '\\') ||
+        (ascii_char == '_') ||
+        (ascii_char == '~'))
+        return true;
+    else
+        return false;
```

Comment:
I believe this could be something like:
```
return ((ascii_char >= '0' && ascii_char <= '9') || 
        (ascii_char >= 'A' && ascii_char <= 'Z') || 
        (ascii_char >= 'a' && ascii_char <= 'z') ||
        (ascii_char == '$') ||
        (ascii_char == '%') ||
        (ascii_char == '&') ||
        (ascii_char == '-') ||
        (ascii_char == '.') ||
        (ascii_char == '/') ||
        (ascii_char == ':') ||
        (ascii_char == '=') ||
        (ascii_char == '?') ||
        (ascii_char == '\\') ||
        (ascii_char == '_') ||
        (ascii_char == '~'));
```

?

Reply:
Yes it's more readable, I also change in the function just below !